cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -28,10 +28,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: fluentd-gcp
-        image: k8s.gcr.io/fluentd-gcp:2.0.16
-        env:
-        - name: FLUENTD_ARGS
-          value: --no-supervisor -q
+        image: gcr.io/stackdriver-agents/stackdriver-logging-agent:{{ fluentd_gcp_version }}
```

Comment:
I believe one important difference between this and the metadata agent config is that the logging agent is sometimes deployed in the non-GKE environment, so this substitution may have to happen even for non-GKE users. Is this something we're worried about?
@kawych, since you have added the original, any comments here? @piosz @x13n @crassirostris may also want to chime in.

Reply:
This substitution needs to happen in https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh in OSS. See e.g. how {{ prometheus_to_sd_prefix }} is substituted below.

Reply:
Ok, I scrolled down and I see it's there and that it has a default, so it should be fine.